nat-lab/tests/test_direct_connection.py

```diff
@@ -32,7 +47,7 @@
         STUN_PROVIDER,
         ConnectionTag.DOCKER_SYMMETRIC_CLIENT_1,
         ConnectionTag.DOCKER_FULLCONE_CLIENT_1,
-        DOCKER_SYMMETRIC_GW_1_IP,
```

Comment:
Is this change really intended? If yes, why should we remove symmetric NAT <-> fullcone NAT test?

Reply:
Oh wait I got confused about arguments, this is reflexive IP, not the NAT type. Ignore this comment.

Reply:
In the test the first client ( Symetric ) will try to ping client two ( Fullcone ) while the second client's reflexive ip path is disable.